modules/olm-operatorhub-architecture.adoc

```diff
@@ -5,15 +5,15 @@
 [id="olm-operatorhub-arch_{context}"]
 = OperatorHub architecture
 
-The OperatorHub UI component is driven by the Marketplace Operator by default on
+OperatorHub's UI component is driven by the Marketplace Operator by default on
```

Comment:
ISG advises to avoid contractions, so I think the way it was before works better...even though I know you're looking to drop the article.

Reply:
"The UI component for OperatorHub", hopefully not too awkward.